cachito/web/api_v1.py

```diff
@@ -75,6 +76,22 @@ def get_request(request_id):
     return flask.jsonify(Request.query.get_or_404(request_id).to_json())
 
 
+@api_v1.route("/requests/<int:request_id>/configuration-files", methods=["GET"])
+def get_request_config_files(request_id):
+    """
+    Retrieve the configuration files associated with the given request.
+
+    :param int request_id: the value of the request ID
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raise NotFound: if the request is not found
+    """
+    config_files = Request.query.get_or_404(request_id).config_files_base64
+    config_files_json = [config_file.to_json() for config_file in config_files]
+    config_files_json = sorted(config_files_json, key=lambda c: c["path"])
+    return flask.jsonify(config_files_json)
```

Comment:
This should just return an empty list if there are no config files, right? Can we add a unit test for it?